xml2kinto/kinto.py

```diff
@@ -1,7 +1,18 @@
-def get_kinto_records(kinto_client, bucket, collection, permissions):
+def get_kinto_records(kinto_client, bucket, collection, permissions,
+                      schema=None):
     """Return all the kinto records for this bucket/collection."""
     # Create bucket if needed
     kinto_client.create_bucket(bucket, if_not_exists=True)
-    kinto_client.create_collection(
+    collection_data = {}
+    if schema is not None:
+        collection_data['schema'] = schema
+
+    previous_collection = kinto_client.create_collection(
         collection, bucket, permissions=permissions, if_not_exists=True)
+
+    if schema and ('schema' not in previous_collection or
+                   previous_collection['schema'] != schema):
+        kinto_client.patch_collection(collection=collection, bucket=bucket,
+                                      data=collection_data)
+
```

Comment:
So all this is for the use case where we update the schema on an existing collection? If so, please 1/ extract that into a helper function 2/ document it (comments and docstring)